feature/stamps/src/main/java/com/momomomo111/stamps/StampsScreen.kt

```diff
@@ -0,0 +1,83 @@
+package com.momomomo111.stamps
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.compose.composable
+import io.github.droidkaigi.confsched2023.ui.SnackbarMessageEffect
+import kotlinx.collections.immutable.ImmutableList
+
+const val stampsScreenRoute = "stamps"
+fun NavGraphBuilder.nestedStampsScreen(
+    onStampsClick: () -> Unit,
+) {
+    composable(stampsScreenRoute) {
+        StampsScreen(
+            onStampsClick = onStampsClick,
+        )
+    }
+}
+
+fun NavController.navigateStampsScreen() {
+    navigate(stampsScreenRoute)
+}
+
+const val StampsScreenTestTag = "StampsScreen"
+
+@Composable
+fun StampsScreen(
```

Comment:
Could you add tests for StampsScreen for improving quality? 🙏 
Since contributors already know the feature so I think it is ok to add screenshot testing
You can refer this https://github.com/DroidKaigi/conference-app-2023/pull/409/files

Reply:
Applied in this commit.

7fe77873e3c11b39916e2933e4b4ac84e469c0db